### ISSUE TITLE
Thank you for this dev

### ISSUE THREAD

**Issue by DealsBeam** [DaniyalAhmadSE/danswriter#2]
As a blogger user, this will be very useful for me, do you still plan to update it? are you willing to accept a donation to assist me in setting it up?

Thank you

**Comment by DaniyalAhmadSE** [comment]
Hi,

Very pleased to know that this can come in handy for you. Apologies for my late response as I'm a final year student and things are quite busy. I can indeed assist you in setting it up over the weekend.

Thank you

**Comment by DaniyalAhmadSE** [comment]
> As a blogger user, this will be very useful for me, do you still plan to update it? are you willing to accept a donation to assist me in setting it up?
> 
> Thank you

Hi,

Very pleased to know that this can come in handy for you. Apologies for my late response as I'm a final year student and things are quite busy. I can indeed assist you in setting it up over the weekend.

Thank you

**Comment by DealsBeam** [comment]
Thank you for the reply, I was not able to set it up because PaLM 2 API no longer exists

**Comment by DaniyalAhmadSE** [comment]
> Thank you for the reply, I was not able to set it up because PaLM 2 API no longer exists



The PaLM API is working for me. It was in early access when I got it and I had to wait a few days for it. I don't know if it's still in early access but it's running. However, as mentioned in the [readme](https://github.com/DaniyalAhmadSE/danswriter?tab=readme-ov-file#) file it doesn't generate as accurately formatted content as you will get with tgpt. So I would recommend you to use that for now. Once I get some time from my schoolwork (by the end of January) I will update it to use more free variants to get better results.

Thank you
